src/Domain/DomainHelpers.cpp

```diff
@@ -990,33 +998,41 @@ cyl_wedge_coordinate_maps(
       } else {
         temp_outer_radius = outer_radius;
       }
+      const auto radial_distribution_this_shell =
+          (shell == radial_partitioning.size() and shell > 0)
```

Comment:
You checked above that `outer_radial_distribution` is correct in the `shell == 0` case, so you don't have to check again here.